### ISSUE TITLE
Information about the source of the forecast

### ISSUE THREAD

**Issue by piec** [open-meteo/open-meteo#35]

**Comment by open-meteo** [comment]
Yes, it was already mentioned in #31. France is mostly covered by ICON D2. Arome might be an option, but it is a pain to process it...

ECMWF has a yearly subscription fee (and broadcast fee) of 180.000 Euro. Also according to their terms, it is not allowed to redistribute ECMWF data again.

**Comment by open-meteo** [comment]
Yes, ICON EU is also in the mix.

Last time I checked, you have to use a job queue system to retrieve each file individually. Processing grib files is fine.
